### DISCUSSION TITLE
Funding Fridays: A Fresh Initiative to Boost OWASP Projects

### DISCUSSION THREAD

**Discussion by ghost** [OWASP/www-committee-project#72] [Collaborative Funding]
Hello, OWASP community!

In our continuous effort to improve, support, and foster growth within the Open Worldwide Application Security Project (OWASP), we're thrilled to announce a new initiative - **Funding Fridays**! This initiative is spearheaded by our dedicated volunteer grant coordinator to create a platform for collaboration, exploration, and application for grants focused on OWASP projects.

## The Vision Behind Funding Fridays

The mission of OWASP is to make software security visible, so that individuals and organizations can make informed decisions about true software security risks. But, we understand that accomplishing this mission often requires funding. Many brilliant ideas and crucial projects might be left unattended due to lack of resources. That's where Funding Fridays come in.

With Funding Fridays, we aim to bridge the gap between grant opportunities and the talented individuals who are working on promising OWASP projects. We want to encourage and support these individuals and teams by providing resources, guidance, and a platform to seek funding that will propel their projects forward.

## What are Funding Fridays?

Every Friday, our community will come together virtually to discuss, brainstorm, and explore grant opportunities. We aim to create an ecosystem where our members can:

1. **Discuss Projects**: Share what you're working on, your plans, needs, and the challenges you're facing.
2. **Explore Grants**: Learn about the different grants available, both within the OWASP Foundation and from external sources.
3. **Craft Grant Applications**: Receive expert advice on how to write compelling grant applications, ensuring the potential and relevance of your project is effectively communicated.

## How Can You Get Involved?

Whether you are working on a new project that needs funding, or you're someone experienced in applying for grants and eager to share your wisdom, we welcome you to join our Funding Fridays.

Here are the steps to get involved:

1. **Sign Up**: To join Funding Fridays, join the #funding channel on Slack  [here](https://join.slack.com/t/owasp/shared_invite/zt-1xc0mkfia-SH4tDusPag0jugVVSggpcA).
2. **Add**: The event to your [calendar](https://calendar.google.com/calendar/embed?src=c_5a0a0a188707b88157512fb81b4b2149d022932f191803f567b09fdd6d777d24%40group.calendar.google.com) and join the Slack Huddle 1PM EST every Friday for 15 minutes.
3. **Prepare**: Come ready with your project details, questions, or experiences. This is a collaborative space where everyone's input is valuable.
4. **Participate**: Engage in the discussion, ask questions, offer advice, and collaborate with your fellow OWASP members.

## Looking Ahead

As we kickstart Funding Fridays, we're excited about the potential of this initiative to not only fuel existing OWASP projects but also to inspire new ones. We believe that by demystifying the grant application process and providing a supportive space for members to collaborate, we can unlock new opportunities and advancements in the field of web application security.

Remember, the success of Funding Fridays relies on the participation of our vibrant OWASP community. So, we encourage you to get involved, share this initiative within your network, and let's work together to make the internet a safer place!

Join us on this journey every Friday and let's create a stronger, safer web, together. Funding Fridays are not just about gaining financial support, but also about building a supportive, innovative community within OWASP. 

We look forward to seeing you at the next Funding Fridays session!

*Blog post written by Donnie Brown, Volunteer Grant Coordinator at OWASP.*

**Comment by northdpole** [comment]
Hey Donnie, this is a great initiative, small nit: the calendar link does not open the event, instead it opens a no-name meeting for some random time on the same day you click it.

I suspect either permission issues or wrong link maybe?

**Reply by ghost** [reply]
Thank you @northdpole - I've updated the link - https://calendar.google.com/calendar/embed?src=c_5a0a0a188707b88157512fb81b4b2149d022932f191803f567b09fdd6d777d24%40group.calendar.google.com
